.github/workflows/ci-build.yml

```diff
@@ -395,17 +399,22 @@ jobs:
 
     - name: Build
       shell: msys2 {0}
+      env:
+        NINJA_STATUS: "[%f/%t -- %e] "
       run: |
         mkdir build
         cd build
 
         echo "::group::CMake"
-        cmake .. -G"MSYS Makefiles"
+        cmake .. \
+          -GNinja \
+          -DCMAKE_CXX_FLAGS="-fuse-ld=lld"
+          # EOF
         echo "::endgroup::"
 
         echo "::group::Build"
         echo "Running on $(nproc) cores"
-        cmake --build . -j $(nproc)
+        cmake --build .
```

Comment:
no more -j? And if that was intentional, please update the `echo` above it too :)